packages/integrations/deno/readme.md

```diff
@@ -0,0 +1,66 @@
+# @astrojs/deno
+
+A server-side rendering adapter for use with Deno targets. Write your code in Astro/Node and deploy to Deno servers.
+
+In your astro.config.mjs use:
+
+```js
+import { defineConfig } from 'astro/config';
+import deno from '@astrojs/deno';
+
+export default defineConfig({
+  adapter: deno()
+});
+```
+
+After performing a build there will be a `dist/server/entry.mjs` module. You can start a server simply by importing this module:
+
+```js
+import './dist/entry.mjs';
```

Comment:
Can we export a function that starts the server? Or, is the idea that this will also support running from `deno run ./dist/entry.mjs`? I like to avoid side-effect imports like this, if we can help it.

Reply:
Ah, nvm i see the mention in the readme below. Interesting that we can support both!

Reply:
The use case for the side-effect version is deploying straight to Deno Deploy without needing to add any code of your own.

Reply:
sgtm